scripts/inclusion-bot.yaml

```diff
@@ -0,0 +1,313 @@
+# How this file works:
+#
+# This is a YAML file. You can read a little about them here:
+# http://thomasloven.com/blog/2018/08/YAML-For-Nonprogrammers/
+#
+# The link property is where the "(What's this?)" links to. It is currently the
+# Internet Archive link of the 18F blog post about the original "guys bot,"
+# since the 18F blog post was removed.
+#
+# The message property is used to describe the purpose of the bot and encourage
+# people to learn more. This is displayed after the list of triggering phrases
+# and suggested alternatives.
+#
+# The triggers property is the list of phrases that will cause the bot to
+# respond, what it should ignore, and alternative words or phrases to suggest.
+#
+# To add an entirely new response to the Inclusion Bot, you will need to add
```

Comment:
I appreciate the way you're encouraging openness here, but perhaps it would be useful to suggest chatting a bit in #g-diversity before making unilateral changes?

Reply:
Added some words about that!